### DISCUSSION TITLE
how to port Linux to esp32

### DISCUSSION THREAD

**Discussion by 1212132** [espressif/arduino-esp32#8905] [Q&A]
I'm a new ass,how to do this?

**Comment by AliMoal** [comment]
Hi, 
Generally, I know Linux has been ported on Xtensa, but for now I guess this repo can help you:
https://github.com/ESP32DE/Boot-Linux-ESP32S3-Playground

**Comment by lucasssvaz** [comment]
A full implementation of Linux for ESP32 is not possible as it doesn't have a MMU, thus not having a way to properly create and manage processes.
